config/application.yml.example

```diff
@@ -46,6 +46,8 @@ PRODUCER_ADDRESS_FORMAT:     '%s-producer.barito-worker.svc:8080'
 KIBANA_ADDRESS_FORMAT:       '%s-kb-http.barito-worker.svc:5601'
 ES_ADDRESS_FORMAT:           '%s-es-http.barito-worker.svc'
 DEFAULT_MAX_TPS:             '5000'
+MONITORING_LINK_FORMAT:      '%s'
```

Comment:
is this env being use in our code? I checked no code called this env

Reply:
it is used here mas https://github.com/BaritoLog/BaritoMarket/blob/46a2d75906f70463e7b8bdca1f36912e8ab1ce4a/app/controllers/app_groups_controller.rb#L41

but it seems there's no env related to that and triggers an error, so we add it to the config